### DISCUSSION TITLE
What did 512x model mean?

### DISCUSSION THREAD

**Discussion by garyakimoto** [d8ahazard/sd_dreambooth_extension#1159] [Q&A]
I have a question want to ask for long time...

I find a option under Dreambooth's create tag, there a cleck box with "512x Model" , is that mean when I create a model > 512 I don't need to check it? If I want to create a model with 768 pixel .. I just need to change Image Processing > Max Resolution to 768
?

Thank you so much

**Comment by SebaSpeed** [comment]
If you are using a 768px model, uncheck the 512x model box.
